AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/releases/ReleasesRest.java

```diff
@@ -47,19 +52,95 @@ public class ReleasesRest {
 
     @GET
     @ApiOperation(value = "Get releases", notes = "Returns all releases")
-    public List<ReleaseEntity> getReleases() {
-        return releaseMgmtService.loadAllReleases(true);
+    public List<ReleaseEntity> getReleases(@DefaultValue("-1") @QueryParam("start") int start, @DefaultValue("-1") @QueryParam("limit") int limit) {
+        if (start < 0 || limit < 0) {
+            return releaseLocator.loadAllReleases(true);
+        } else {
+            return releaseLocator.loadReleasesForMgmt(start, limit, true);
+        }
     }
 
     @GET
     @Path("/{id}")
     @ApiOperation(value = "Get a release", notes = "Returns the specifed release")
     public Response getRelease(@PathParam("id") int id) {
         ReleaseEntity release = releaseLocator.getReleaseById(id);
-        if(release == null) {
+        if (release == null) {
             return Response.status(Response.Status.NOT_FOUND).build();
         }
 
         return Response.ok(release).build();
     }
+
+
+    @GET()
+    @Path("/count")
+    @ApiOperation(value = "Get the number of releases", notes = "Returns the total amount of release entities")
+    public int getCount() {
+        return releaseLocator.countReleases();
+    }
+
+
+    @GET()
+    @Path("/default")
+    @ApiOperation(value = "Get default release", notes = "Returns the default release entity")
+    public ReleaseEntity getDefaultRelease() {
+        return releaseLocator.getDefaultRelease();
+    }
+
+    @POST
+    @ApiOperation(value = "Add a release")
+    public Response addRelease(@ApiParam() ReleaseEntity request) {
+        if (request.getId() != null) {
```

Comment:
Same as above - use exception mappers to set the proper error response code instead of individually checking error cases and set the response codes manually.